wikichron/dash/main.py

```diff
@@ -651,4 +681,4 @@ def update_graph(scale_type, slider, selection_json):
                 height=450,
                 hovermode='closest'
             )
-        }
+        }
```

Comment:
Look for an option in your editor because it should have one that forces or adds a new line at the end of every file.